exp/README.md

```diff
@@ -0,0 +1,46 @@
+# Benchmark on MLE-Bench
+
+## Installation
+
+You need to install [Git LFS](https://git-lfs.github.com/) to set up, and your Python version must be >= 3.11.
+
+```shell
+GIT_LFS_SKIP_SMUDGE=1 pip install -e .[bench]
+
+# In Windows CMD
+set GIT_LFS_SKIP_SMUDGE=1
+pip install -e .[bench]
+
+# In Windows PowerShell
+$env:GIT_LFS_SKIP_SMUDGE=1
+pip install -e .[bench]
+```
+
+Then run the following command to set up the MLE-Bench:
+```shell
+mle-exp init
+```
+
+## Usage
+
+- Prepare the dataset:
+```shell
+# Prepare all dataset
+mle-exp prepare --all
```

Comment:
lets move this to the bottom.

We can have 2 sections 

Lite
And
Full

Now we just focus the Full